internal/sdkprovider/service/kafkatopic/kafka_topic_create.go

```diff
@@ -25,6 +27,18 @@ func (w *kafkaTopicCreateWaiter) RefreshFunc() resource.StateRefreshFunc {
 	// Should check if topic does not exist before create
 	// Assumes it exists, should prove it doesn't by getting no error
 	return func() (interface{}, string, error) {
+		defer func() { w.c++ }()
+
+		if w.c == 0 {
+			topic, err := w.Client.KafkaTopics.Get(w.Project, w.ServiceName, w.CreateRequest.TopicName)
```

Comment:
This will slow down everything. Currently (in main) we use bulk endpoints, but this will make all topics (10-100-1000) make Get before create.
Please use bulk endpoints.